provider/cmd/crd2pulumi/gen/schema.go

```diff
@@ -187,27 +177,6 @@ func addType(schema map[string]interface{}, name string, types map[string]pschem
 	}
 }
 
-// UnderscoreFields replaces the hyphens in the x-kubernetes-... fields with
-// underscores
-func UnderscoreFields(schema map[string]interface{}) {
```

Comment:
This is no longer needed?

Reply:
For this commit I temporarily stopped generating `new CustomResourceDefinition()...` helper classes in TypeScript, so this function wasn't needed. I'll add back in again, but once `kube2pulumi` fixes CRDs, this will be removed permanently.